utils/yaml-docker-execute.sh

```diff
@@ -31,22 +31,23 @@ _ASH_UTILS_LOCATION=${_ASH_UTILS_LOCATION:-/utils}
 _ASH_CFNRULES_LOCATION=${_ASH_CFNRULES_LOCATION:-/cfnrules}
 _ASH_RUN_DIR=${_ASH_RUN_DIR:-/run/scan/src}
 
+source ${_ASH_UTILS_LOCATION}/common.sh
+
 #
 # Allow the container to run Git commands against a repo in ${_ASH_SOURCE_DIR}
 #
-git config --global --add safe.directory ${_ASH_SOURCE_DIR} >/dev/null 2>&1
-git config --global --add safe.directory ${_ASH_RUN_DIR} >/dev/null 2>&1
+git config --global --add safe.directory "${_ASH_SOURCE_DIR}" >/dev/null 2>&1
+git config --global --add safe.directory "${_ASH_RUN_DIR}" >/dev/null 2>&1
 
 # cd to the source directory as a starting point
-cd ${_ASH_SOURCE_DIR}
-# Check if the source directory is a git repository and clone it to the run directory
-if [[ "$(git rev-parse --is-inside-work-tree 2>/dev/null)" == "true" ]]; then
-  if [[ "$_ASH_EXEC_MODE" != "local" ]]; then
-    git clone --depth=1 --single-branch ${_ASH_SOURCE_DIR} ${_ASH_RUN_DIR} >/dev/null 2>&1
-  fi
-  _ASH_SOURCE_DIR=${_ASH_RUN_DIR}
-  cd ${_ASH_RUN_DIR}
-fi;
+cd "${_ASH_SOURCE_DIR}"
+debug_echo "[yaml] pwd: '$(pwd)' :: _ASH_SOURCE_DIR: ${_ASH_SOURCE_DIR} :: _ASH_RUN_DIR: ${_ASH_RUN_DIR}"
+# # Check if the source directory is a git repository and clone it to the run directory
```

Comment:
Should we remove this code instead of commenting it out?

Reply:
@rafaelpereyra yeah, that's a fair callout. I'll do that, along with the other instances across the other scripts where it was commented out the same.

Reply:
@scrthq  - while you're in all the `.sh` files, do another sweep/check for unquoted `${_ASH_SOURCE_DIR}`.

Reply:
All resolved and cleaned up